internal/pkg/generator/forwarder/generator.go

```diff
@@ -61,9 +61,24 @@ func Generate(collectionType logging.LogCollectionType, clfYaml string, includeD
 		}
 	}
 
-	spec, status := clRequest.NormalizeForwarder(map[string]bool{})
-	log.V(2).Info("Normalization", "spec", spec)
-	log.V(2).Info("Normalization", "status", status)
+	// Added because this generator is used for tests and the tests assume a correct
+	// CLF spec.
+	// Originally, this generator normalized the forwarder spec only which dropped
+	// input specs with names that were reserved (application, audit, infrastructure).
+	// With the change in validation, CLFs are rejected outright with input specs name equal to reserved names.
```

Comment:
Does looks to break input specs with a admin defined named where for instance you need or can "create" an input for selectively collecting a specific set of namespaces?

Reply:
I'm not reading the code correctly; we are dropping inputs with reserved words.